src/engine.cc

```diff
@@ -216,20 +234,12 @@ SearchLimits EngineController::PopulateSearchLimits(
     total_moves_time -= time_to_squander;
   }
 
-  constexpr int kSmartPruningToleranceMs = 200;
-  float this_move_weight = ComputeMoveWeight(
-      ply, time_curve_peak, time_curve_left_width, time_curve_right_width);
-  float other_move_weights = 0.0f;
-  for (int i = 1; i < movestogo; ++i)
-    other_move_weights +=
-        ComputeMoveWeight(ply + 2 * i, time_curve_peak, time_curve_left_width,
-                          time_curve_right_width);
-  // Compute the move time without slowmover.
-  float this_move_time = total_moves_time * this_move_weight /
-                         (this_move_weight + other_move_weights);
+  // Evenly split total time between all moves.
+  float this_move_time = total_moves_time / movestogo;
 
   // Only extend thinking time with slowmover if smart pruning can potentially
   // reduce it.
+  constexpr int kSmartPruningToleranceMs = 200;
```

Comment:
I'm not sure anymore that kSmartPruningToleranceMs is a good idea, maybe remove it?

Reply:
I agree, but I think that smart pruning modifications should be separate PR and will need additional testing. Not sure if it can be removed entirely, it seems maybe it should be a function of time budget. We can afford to have it be a second or two in 90 minute games.